apps/ejabberd/src/ejabberd_service.erl

```diff
@@ -348,12 +348,15 @@ code_change(_OldVsn, StateName, StateData, _Extra) ->
 %%          {stop, Reason, NewStateData}
 %%----------------------------------------------------------------------
 handle_info({send_text, Text}, StateName, StateData) ->
+    ?ERROR_MSG("{service:send_text, Text}: ~p~n", [{send_text, Text}]), % is it ever called?
```

Comment:
Maybe `WARNING_MSG` would be just enough. I don't know if this is ever called. Our coverage stats may help you, did you check them?